File: source/manual/ds_writing/writing/steppermotor6.cpp
void DocDs::get_device_property()
{
	Tango::DbData	data;
	data.push_back(DbDatum("Max"));
	data.push_back(DbDatum("Min"));

	get_db_device()->get_property(data);

	if (data[0].is_empty()==false)
		data[0]  >>  max;
	if (data[1].is_empty()==false)
		data[1]  >>  min;
}

File: ds_writing/writing_java/StepperMotor6.java
void public get_device_property() throws DevFailed
{
	String[] prop_names = {"Max","Min"};

	DbDatum[] res_value = db_dev.get_property(prop_names);

	if (res_value[0].is_empty() == false)
		max = res_value[0].extractInt();
	if (res_value[1].is_empty() == false)
		min = res_value[1].extractInt();
}
